apps/mobile/src/features/send/send-form/validation/btc.validation.ts

```diff
@@ -13,7 +13,6 @@ export function formatBitcoinError(errorMessage: BitcoinErrorKey) {
     }),
     InsufficientAmount: t({
       id: 'bitcoin-error.insufficient-amount',
-      // TODO test this - avoids useLingui
       message: `Minimum is ${minimumBtcSpend}`,
```

Comment:
I confirmed this works 
![Screenshot 2025-02-26 at 14 23 27](https://github.com/user-attachments/assets/bc0d2d9d-adae-4ab2-b723-9b15cc59c9a8)